netstandard/ref/System.Reflection.Emit.cs

```diff
@@ -39,6 +49,9 @@ public sealed partial class ConstructorBuilder : System.Reflection.ConstructorIn
         public override System.Reflection.Module Module { get { throw null; } }
         public override string Name { get { throw null; } }
         public override System.Type ReflectedType { get { throw null; } }
+//      [System.ObsoleteAttribute("This property has been deprecated. http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
Is it useful to maintain the attribute in commented form? If it makes diffs simpler, that's fine - I'm not sure how useful it is otherwise. (Unless it also serves as the reason for it to not be included? That would make sense.)

Reply:
Commented APIs should have a justification. It seemed to me the obsolete attribute was as good as any other justification but was already there :-)